packages/atomic/cypress/e2e/refine-toggle.cypress.ts

```diff
@@ -111,6 +149,25 @@ describe('Refine Toggle Test Suites', () => {
         .should('have.length', allFacets.length);
     });
 
+    it('should display the facets in the correct order', () => {
```

Comment:
We are expecting the original order to be preserved even if the facets are wrapped in a facet manager?

Reply:
I think its more that when you resize the page from desktop to mobile, the order is kept the same.